x-pack/solutions/search/test/functional_search/config/config.nav_v2.ts

```diff
@@ -0,0 +1,24 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+
+import type { FtrConfigProviderContext } from '@kbn/test';
+
+export default async function ({ readConfigFile }: FtrConfigProviderContext) {
+  const baseConfig = await readConfigFile(require.resolve('../config'));
+
+  return {
+    ...baseConfig.getAll(),
+    kbnTestServer: {
+      ...baseConfig.get('kbnTestServer'),
+      serverArgs: [
+        ...baseConfig.get('kbnTestServer.serverArgs'),
+        '--feature_flags.overrides.core.chrome.projectSideNav=v2',
```

Comment:
we have config's for features that are currently feature flagged in both serverless and stateful. We _could_ add these tests there instead of introducing a new config. But the config also gives isolation 🤷‍♂️ so I'm ok with it as-is